src/main/java/org/jabref/gui/importer/GenerateEntryFromIdAction.java

```diff
@@ -56,15 +59,18 @@ public void execute() {
         backgroundTask.onFailure((exception) -> {
             String fetcherExceptionMessage = exception.getMessage();
 
-            boolean addEntryFlag;
+            String msg;
             if (exception instanceof FetcherClientException) {
-                addEntryFlag = dialogService.showConfirmationDialogAndWait(Localization.lang("Failed to import by ID"), Localization.lang("Bibliographic data not found. Cause is likely the client side. Please check connection and identifier for correctness.") + "\n" + fetcherExceptionMessage, Localization.lang("Add entry manually"));
+                msg = Localization.lang("Bibliographic data not found. Cause is likely the client side. Please check connection and identifier for correctness.") + "\n" + fetcherExceptionMessage;
             } else if (exception instanceof FetcherServerException) {
-                addEntryFlag = dialogService.showConfirmationDialogAndWait(Localization.lang("Failed to import by ID"), Localization.lang("Bibliographic data not found. Cause is likely the server side. Please try agan later.") + "\n" + fetcherExceptionMessage, Localization.lang("Add entry manually"));
+                msg = Localization.lang("Bibliographic data not found. Cause is likely the server side. Please try agan later.") + "\n" + fetcherExceptionMessage;
```

Comment:
I notice a typo "agan" instead of again. Can you please correct that in the l10n as well?

Reply:
Sure 👍